src/runtime/cli/kata-check.go

```diff
@@ -71,6 +75,15 @@ const (
 	genericCPUFlagsTag    = "flags"      // nolint: varcheck, unused, deadcode
 	genericCPUVendorField = "vendor_id"  // nolint: varcheck, unused, deadcode
 	genericCPUModelField  = "model name" // nolint: varcheck, unused, deadcode
+
+	// A release URL is expected to be prefixed with this value
+	projectAPIURLPrefix = "https://api.github.com/repos/kata-containers"
+
+	// URL used to determine latest release
+	defaultReleaseURL = projectAPIURLPrefix + "/kata-containers/releases/latest"
```

Comment:
Marking as dnm for now since this isn't reliable - the "latest" release is simply the most recently uploaded release! The correct solution is to look through all releases, sort by semver and find the newest from that.

Thanks @jcvenegas for pointing this out!